### DISCUSSION TITLE
What is "Diagram generation settings folder"?

### DISCUSSION THREAD

**Discussion by doktorjw** [hallvard/plantuml#148] [Q&A]
I have the need for a diagram over 4k in size (yep, have a big one as I'm combing through this project I've inherited). From the docs, I can see I need to set an environment variable (set PLANTUML_LIMIT_SIZE=8192, for example).

In the settings for the plantuml plugin, there is a "Diagram generation settings folder" but I can't find any documentation for it anywhere. Is there a way to use this for settings so I can easily set an environment variable for the sizing of the generated diagram? What is that folder used for, and how is it used?

**Comment by hallvard** [comment]
The idea is to load various kinds of settings from one of more files (.properties or other) in this folder, rather than using Eclipse properties and making a UI for each one of them. The kinds of settings I envision are those controlling diagram generation, (auto-injected) styling, env. properties for image generation etc. But so far I've been working on the mechanism and haven't defined them. The one you mention is a candidate I will consider!

**Reply by VerifyVenus** [reply]
Has this feature been implemented yet? Also in a position where I need to set PLANTUML_LIMIT_SIZE=8192
